src/compressed_tensors/quantization/lifecycle/initialize.py

```diff
@@ -79,6 +94,8 @@ def initialize_module_for_quantization(
         # no scheme passed and layer not targeted for quantization - skip
         return
 
+    _clear_all_qparams(module)
```

Comment:
I am more in favour of simply erroring out as opposed to clearing it out now

Reply:
I agree we have to be careful since this is an invasive operation, but `initialize_module_for_quantization` only gets called during `apply_quantization_config` and `CompressedLinear.from_linear` (and tests), so I think this would be ok. Erroring out would cause multi-modifier flows targeting the same modules to error out, which is a desired pathway. I think the `initialize` in the name also indicates this.

Perhaps I'm misreading, I'm not sure what you mean by in favor generally speaking, but preferring erroring out to clearing out as it currently is. Can we spend some time in deep dive to discuss?